tgui/packages/tgui/interfaces/PreferencesMenu/preferences/features/game_preferences/hotkeys.tsx

```diff
@@ -1,8 +1,24 @@
-import { CheckboxInputInverse, FeatureToggle } from '../base';
+import { CheckboxInputInverse, createDropdownInput, FeatureToggle, Feature } from '../base';
 
 export const hotkeys: FeatureToggle = {
   name: 'Classic hotkeys',
   category: 'GAMEPLAY',
   description: 'When enabled, will revert to the legacy hotkeys, using the input bar rather than popups.',
   component: CheckboxInputInverse,
 };
+
+export const zone_select: Feature<string> = {
+  name: 'Bodyzone Targetting Mode',
+  category: 'GAMEPLAY',
+  description:
+    'When set to simplified, the bodyzone system will be replaced with a grouped system where you can target legs, arms or body/head. This is useful if you do not have a numpad or want an easier to use system.',
+  component: createDropdownInput(
```

Comment:
This doesn't need to be a dropdown at all, it can just be dual buttons. Dropdowns are bad UX especially if you only have two choices. You also disabled the arrow buttons which makes it even less usable.

Reply:
I don't think dropdowns are bad UX, the reason I chose that is because the dual buttons choices are a bit of a mess in the UI and can span multiple rows. With only 2 buttons it may also be confusing as to which one is selected, tick boxes are ideal for 2 options however in this case the values are used to indicate the function of the targeting mode. I disabled the buttons specifically because there is no logical ordering for targeting modes, so there is no need to have a button that goes to the next one as the user won't know what option is the next one. Personally I think that all of the button options should be made to dropdowns instead, but I will change it for consistency sake.

![image](https://github.com/BeeStation/BeeStation-Hornet/assets/26465327/2cffe449-645c-4dd6-9ad7-d2cef66f459c)

Reply:
It's about information availability and clicks required.

A dropdown requires two clicks minimum, a dropdown with button requires 1 but doesn't give you any information on the other options without an additional click.

A simple list of buttons tells you all the options, they are all accessible in one click. and if you are at all familiar with the UI conventions you know that green is selected (as in every other part of the UI, including your character select). It is very quickly learned.

It may take slightly more vertical space but it's easier to use at a glance.